test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Metadata/ODataUriResolverTests.cs

```diff
@@ -139,33 +139,19 @@ public override void PromoteBinaryOperandTypes(BinaryOperatorKind binaryOperator
         [Fact]
         public void TestStringAsEnumInFunctionParameter()
         {
-            this.TestStringAsEnum(

```

Comment:
Why is this test being removed?

Reply:
With the change in this PR String to Enum Conversion is supported. The two tests here check for error messages on converting a string to Enum. So they are no longer valid. So I am removing them and adding a test to verify the conversion.